docs/Project-Configuration/Evergreen-Data-for-Analytics.md

```diff
@@ -58,6 +58,151 @@ Daily aggregated statistics for test executions run in Evergreen. Test stats are
 | total\_pass\_duration\_ns | DOUBLE  | Total duration, in nanoseconds, of passing tests.
 | task\_create\_iso         | VARCHAR | Date, in ISO format `YYYY-MM-DD`, on which the tests ran.
 
+### Evergreen Finished Tasks
+Finished Evergreen tasks, partitioned by the project and date (in ISO format). For example, the partition `mongodb-mongo-master/2022-12-05` would contain all tasks that completed on `2022-12-05` in the `mongodb-mongo-master` project. When running queries against this view it is highly recommended to always filter by project and date.
+
+#### Table
+| Catalog        | Schema          | View                                    |
+| ---------------|-----------------|-----------------------------------------|
+| awsdatacatalog | dev\_prod\_live | v\_\_evergreen\_\_finished\_tasks\_\_v0 |
+
+#### Columns
```

Comment:
Looks like you are missing the `tags` column.

Reply:
fixed